cookbook/web_server/built_in.rst

```diff
@@ -41,6 +41,15 @@ can change the socket passing an IP address and a port as a command-line argumen
 
 .. note::
 
+    You can use the ``--force`` option to force the web server start
+    even if a web server is already running.
```

Comment:
I think we should reword this a bit. The issue with the current wording is that you may be tempted to try to use the option even when a web server instance is already running which in turn will lead to errors.